cirq/ops/parity_gates_test.py

```diff
@@ -29,7 +29,7 @@
 )
 def test_eigen_gates_consistent_protocols(eigen_gate_type):
     cirq.testing.assert_eigengate_implements_consistent_protocols(
-            eigen_gate_type)
+        eigen_gate_type, ignoring_global_phase=True)
```

Comment:
Can you add a global phase gate to the decomposition instead?

Reply:
Done.